styles/styles.css

```diff
@@ -0,0 +1,202 @@
+/* General Styling */
```

Comment:
I like that you put all of your general styling at the top. This definitely is something I didn't do/wanted to come back to, and it makes it way easier to read.